python/pyhail/context.py

```diff
@@ -278,6 +278,61 @@ def import_plink(self, bed, bim, fam, npartitions=None, delimiter='\\\\s+', miss
         """
         Import PLINK binary file (.bed, .bim, .fam) as VariantDataset
 
+        **Examples**
+
+        Import data from a PLINK binary file
+
+        >>> vds = (hc.import_plink(bed="data/test.bed",
+        >>>                        bim="data/test.bim",
+        >>>                        fam="data/test.fam"))
+
+        Phenotype is quantitative (column 6 in the .fam file)
```

Comment:
I'd make the description self-contained, and not reply on the previous example.  I think just one of these examples is enough.